Quadriga/src/main/java/edu/asu/spring/quadriga/web/search/TextConceptSearchController.java

```diff
@@ -53,85 +52,97 @@
     private IJsonCreator jsonCreator;
 
     @RequestMapping(value = "search/texts")
-    public String search(@RequestParam(defaultValue = "") String conceptId, Model model) throws Exception {
-
-        String conceptUri = conceptId;
-
-        if (!conceptId.isEmpty()) {
-            IConcept concept = cpCache.getConceptByUri(conceptId);
-            
-            if (concept != null) {
-                concept.setId(conceptUri);
-            }
+    public String search(@RequestParam(value = "conceptid1", defaultValue = "") String conceptId1,
+            @RequestParam(value = "conceptid2", defaultValue = "") String conceptId2, Model model) throws Exception {
+
+        ArrayList<String> searchedConcepts = new ArrayList<>();
+        List<List<ITextFile>> texts = new ArrayList<List<ITextFile>>();
+        List<List<String>> handles = new ArrayList<List<String>>();
+        List<String> conceptUriSearchList = new ArrayList<String>();
+        List<IConcept> concepts = new ArrayList<IConcept>();
+        searchedConcepts.add(conceptId1);
+        searchedConcepts.add(conceptId2);
+        for (String conceptUri : searchedConcepts) {
```

Comment:
this looks like is just search for statemnets that have either concept 1 or concept 2 in them, but it should search for statements that have concept 1 **and** concept 2.

Reply:
The for loop is used to collect information about the searched concepts. The actual search happens in 
ITransformedNetwork transformedNetwork = transformationManager
                .getSearchTransformedNetworkMultipleProjects(projectIds, conceptUriSearchList, INetworkStatus.APPROVED); where conceptUriSearchList is a list of the searched concepts and projectIds is the list of public project ids

Reply:
this might only show all statements that have both concepts in them, but it's still showing all texts that have either concept 1 or concept 2 in them, doesn't it?